presentation/feeddiary/src/main/java/com/hilingual/presentation/feeddiary/FeedDiaryScreen.kt

```diff
@@ -123,6 +124,7 @@ private fun FeedDiaryScreen(
     var isUnpublishDialogVisible by remember { mutableStateOf(false) }
 
     var isReportBottomSheetVisible by remember { mutableStateOf(false) }
+    var isReportConfirmDialog by remember { mutableStateOf(false) }
```

Comment:
p2
```suggestion
    var isReportConfirmDialogVisible by remember { mutableStateOf(false) }
```
얘두 그동안 작업했던 네이밍 방식과 비슷하게 수정하는 건 어떨까요..??
(일기 피드백에서는 다 Visible 붙여서 작업해뒀길래 물어봅니당)